.github/workflows/ci.yml

```diff
@@ -36,28 +36,24 @@ jobs:
       - name: Install Rust
         run: rustup update stable
       - run: rustup target add ${{ matrix.target }}
-      # TODO: Currently, valuable cannot build without `alloc` feature
-      #       because `Debug` impl of `dyn Enumerable` uses `format!` macro.
-      # - run: cargo build --target ${{ matrix.target }} --no-default-features
-      #   working-directory: valuable
-      # - run: cargo build --target ${{ matrix.target }} --no-default-features --features derive
-      #   working-directory: valuable
+      - run: cargo build --target ${{ matrix.target }} --no-default-features
+        working-directory: valuable
+      - run: cargo build --target ${{ matrix.target }} --no-default-features --features derive
+        working-directory: valuable
       - run: cargo build --target ${{ matrix.target }} --no-default-features --features alloc
         working-directory: valuable
       - run: cargo build --target ${{ matrix.target }} --no-default-features --features alloc,derive
         working-directory: valuable
 
-  # TODO: Currently, valuable cannot build without `alloc` feature
-  #       because `Debug` impl of `dyn Enumerable` uses `format!` macro.
-  # features:
-  #   runs-on: ubuntu-latest
-  #   steps:
-  #     - uses: actions/checkout@v2
-  #     - name: Install Rust
-  #       run: rustup update stable
-  #     - name: Install cargo-hack
-  #       run: cargo install cargo-hack
-  #     - run: cargo hack build --workspace --feature-powerset --no-dev-deps
+  features:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Rust
+        run: rustup update stable
+      - name: Install cargo-hack
+        run: cargo install cargo-hack
+      - run: cargo hack build --workspace --feature-powerset --no-dev-deps
```

Comment:
Oh, our MSRV is 1.51 (because we use min const generics), so we can use [`resolver = "2"`](https://doc.rust-lang.org/cargo/reference/features.html#feature-resolver-version-2) that the more preferable than --no-dev-deps ([that edit Cargo.toml](https://github.com/taiki-e/cargo-hack/issues/15)).